src/components/dom.jsx

```diff
@@ -59,6 +60,10 @@ const DOM = (props) => {
 		escapedState,
 	};
 
+	if(clientFilename){
```

Comment:
just catching up on mup-web PR - I see that you're intending `scripts` to be the single source of truth for the scripts to render. That makes sense - the thing I'd recommend here is throwing an error like

```js
if (!scripts.length) {
  throw new Error('No client scripts supplied - check the `makeRenderer` call in your server entry script');
}
```